import_schemas.md

```diff
@@ -0,0 +1,154 @@
+## Import Schema
```

Comment:
I'm not sure if it makes more sense to document the import schema or the export schema or both or neither. They're kind of documented implicitly in each `config` detail file. Kind of open to anything on this one.

Reply:
yeah. they felt pretty self documenting inside of the code. the export schema is probably more important. i'll  remove the import schema info and put a nearly blank doc the coloumn data i guess.

that was part of https://app.asana.com/0/1202413225616901/1202444697182110/f.